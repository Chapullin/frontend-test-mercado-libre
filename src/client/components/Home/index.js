import React from 'react';

export default function () {
  return (
    <div>
      <h1>Busca el articulo que quieras</h1>
    </div>
  );
}
