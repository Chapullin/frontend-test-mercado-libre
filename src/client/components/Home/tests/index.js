import React from "react";
import renderer from "react-test-renderer";
import Home from "../../Home";
import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";

describe("Home", () => {
  it("should match snapshot - base", () => {
    const component = renderer
      .create(
        <Home />
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
