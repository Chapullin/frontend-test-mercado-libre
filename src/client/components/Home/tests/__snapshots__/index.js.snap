// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot - base 1`] = `
<div>
  <h1>
    Busca el articulo que quieras
  </h1>
</div>
`;
